pkg/rbac/rbac.go

```diff
@@ -137,6 +141,11 @@ func NewEnforcer(ctx context.Context, kubeClient *kubernetes.Kubernetes, l *zap.
 	if err != nil {
 		return nil, err
 	}
+	cm, err := adapter.ConfigMap(ctx)
+	if err != nil {
+		return nil, errors.Join(err, errors.New("failed to get RBAC ConfigMap"))
+	}
+	enforcer.EnableEnforce(IsEnabled(cm))
```

Comment:
Just to be sure, this will affect all of our custom validations because we always use the `can...` functions that call the enforcer, correct? Is there any custom validation that doesn't rely on the enforcer under the hood?

Reply:
There isn't.. Yes this will disable the enforcer, so even the custom validations will effectively be ignored